lnt/server/ui/views.py

```diff
@@ -186,6 +186,23 @@ def ts_data(ts):
     }
 
 
+def determine_aggregation_function(function_name):
+    """
+    Return the aggregation function associated to the provided function name. This is used by
+    dropdown menus that allow selecting from multiple aggregation functions.
+    """
+    if function_name == 'min':
+        return lnt.util.stats.safe_min
+    elif function_name == 'max':
+        return lnt.util.stats.safe_max
+    elif function_name == 'mean':
+        return lnt.util.stats.mean
+    elif function_name == 'median':
+        return lnt.util.stats.median
+    else:
+        assert False, f'Invalid aggregation function name {function_name}'
```

Comment:
since this is parsing user input (via the url's query params), this should probably be an `abort(400, f'Invalid aggregation function name {function_name}')`, rather than an `assert`.

Reply:
or this should return `None`, and the caller should do that... whatever.

Reply:
That's right -- I hadn't thought that users could craft an URL of their choosing, but it makes sense. So the aggregation function is not something that has an internal contract to the code base, it must be considered user input. Will switch to 404 somehow.